scripts/src/3rdparty.lua

```diff
@@ -1514,8 +1518,6 @@ end
 	}
 	if _OPTIONS["targetos"]=="macosx" then
 		files {
-			MAME_DIR .. "3rdparty/bgfx/src/glcontext_eagl.mm",
-			MAME_DIR .. "3rdparty/bgfx/src/glcontext_nsgl.mm",
 			MAME_DIR .. "3rdparty/bgfx/src/renderer_mtl.mm",
```

Comment:
Will this potentially cause issues for older Apple hardware that has poor Metal support?

Reply:
Probably, but OpenGL support was dropped from upstream bgfx:
https://github.com/bkaradzic/bgfx/commit/928800feab95d7bc3f4680d91c3e4754c852d783

If the PR is interesting but we want to keep OpenGL on Macs, I can see if the necessary commits can be cherry-picked instead.

Reply:
@rb6502 how many people are going to want my head if bgfx via OpenGL is no longer possible on macOS?

Reply:
I just got done promising Mac users I wouldn't lift the system requirements until January, so I would strongly prefer to hold off on anything that would affect that until then.

Reply:
Particularly as Belegdol says this doesn't actually work that well (which describes the totality of Wayland in my experience, to be fair).

Reply:
Fair enough. Maybe we can leave this PR open in the draft state as a proof-of-concept?